.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -8,6 +8,7 @@ jobs:
       win:
         CONFIG: azure-win-64
         CF_MAX_PY_VER: 37
+        CONDA_BLD_PATH: D:\bld
```

Comment:
@mariusvniekerk, can these changes be merged into master? Or should they be removed before merging?

Reply:
Ping...